### PR TITLE
Change position to a struct of two values

### PR DESCRIPTION
**Pull request by Jackojc**

### CODE REVIEW

File: token.h [hunk 1]
```diff
@@ -10,16 +10,27 @@
...
 
 namespace ws::token {
-    using Type     = std::uint16_t;
-    using Position = std::uint32_t;
+    using Type = std::uint16_t;
+
+    struct Position {
+        std::uint32_t line = 1, column = 1;
+
+        void ln(unsigned n = 1) {
```
[the remark this answers is not in the document]
Comment (Hazurl) [reply token.h]: col could be `next` and `ln` be `nextln` ?

**Comment by Jackojc** [comment]
@Hazurl fixed

File: token.h [hunk 2]
```diff
@@ -13,14 +13,17 @@ namespace ws::token {
...
 
     struct Position {
-        std::uint32_t line = 1, column = 1;
+        using size_type = std::uint32_t;
 
-        void ln(unsigned n = 1) {
+
+        size_type line = 1, column = 1;
+
+        void nextline(size_type n = 1) {
```
Comment (Hazurl) [inline token.h]: next_line ?

Comment (Jackojc) [reply token.h]: i think its ok

Comment (Jackojc) [reply token.h]: ill change to nextln

**Pull request merged**